product-service/src/main/java/com/extremecoder/productservice/dto/Response.java

```diff
@@ -0,0 +1,26 @@
+package com.extremecoder.productservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+import java.util.List;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Response {
+    private Long timeStamp;
+    private int statusCode;
```

Comment:
Please don't use primitive type here. Instead use Object type.

Reply:
Resolved it. Thanks.